include/unifex/just.hpp

```diff
@@ -79,7 +79,8 @@ class _sender<Values...>::type {
   static constexpr bool sends_done = false;
 
   template(typename... Values2)
-    (requires (sizeof...(Values2) == sizeof...(Values)))
+    (requires (sizeof...(Values2) == sizeof...(Values)) &&
```

Comment:
Can you use `AND` here instead of `&&` so that it does the lazy thing with the SFINAE versions of the macros?